### ISSUE TITLE
Unable to install Patch Kexts

### ISSUE THREAD

**Issue by Wh00ps13** [BenSova/Patched-Sur#730]
**Describe the bug**
Unable to install Patch Kexts

**To Reproduce**
1. Upgrade patch sure to 1.0.0
2. Update using "Run Updates"
3. Wait until ready
4. Open Patch Sur and click "Patch Kexts"

```
Error 1x127
Message: "/usr/local/lib/Patched-Sur-Patches/Scripts/PatchSystem.sh: line 56: /usr/local/lib/Patched-Sur-Patches/Scripts/../NeededPatches.sh: No such file or directory"
Output: "Checking environment...
[INFO] We're booted into full macOS.
[INFO] Using dirname source.

Confirming patch location...
Set to detect patches, restarting PatchSystem with NeededPatches..."
```

**Desktop (please complete the following information):**
 - Upgrading from: [e.g. Big Sur v11.5]
- Upgrading to: [e.g. Big Sur v11.5.2]
 - Mac Model: [e.g. 13" 2012 MacBook Pro]
 - Version [e.g. v1.0.0]

**Comment by Monkiey** [comment]
try this terminal command:PSPATH=~/.patched-sur
cd $PSPATH && rm -rf Patched-Sur-Patches*
curl -Lo Patched-Sur-Patches.zip https://codeload.github.com/BenSova/Patched-Sur-Patches/zip/8c27158
cd /usr/local/lib && sudo rm -rf Patched-Sur-Patches*
sudo unzip -q $PSPATH/Patched-Sur-Patches.zip
sudo mv Patched-Sur-Patches-8c27158 Patched-Sur-Patches

**Comment by BenSova** [comment]
Duplicate issue.
